components/automate-ui/src/app/pages/+compliance/+reporting/reporting-searchbar/reporting-searchbar.component.ts

```diff
@@ -45,7 +46,7 @@ export class ReportingSearchbarComponent implements OnInit {
   filtersVisible = true;
   suggestionsVisible = false;
   isLoadingSuggestions = false;
-  visibleDate;
+  visibleDate: moment.Moment;
```

Comment:
Switch from a Date object to a Moment object, because with a Moment we can keep the timezone in UTC.